Code/Code_Dev/taxonomizer.jl

```diff
@@ -45,7 +45,7 @@ function taxonomizer(df::DataFrame, type::Symbol=:hosts; names::Symbol=:Name)
 
     # GO BR
     for i in 1:n
-        @info synonyms[i]
+        # @info synonyms[i]
```

Comment:
dropped print statement to speed up runs/make logs more readable

Reply:
i think they can/should be dropped entirely?